### ISSUE TITLE
UI: Distinguish between Twitter messages and Instagram messages

### ISSUE THREAD

**Issue by jamiesnape** [XDATA-Year-3/geoapp#163]
This was mostly in the list of messages. Maybe there could be an image icon next to messages with images attached. Maybe the option to show only points with images on the map, but seemed less of a priority,

**Comment by manthey** [comment]
Was there an actual use case for distinguishing between the two, or was it just a desire (I ask to help prioritize features)?  Or is it a desire to know which messages have images (since some twitter messages have images)?

Since we pull images from instagram, and some images are have been deleted, the only way to really know if an image is available is to attempt to fetch it (or prefetch them ahead of time).

**Comment by jamiesnape** [comment]
Yes. Say you wanted to see all the images taken at the finish line of the marathon because there was an incident there.

**Comment by jamiesnape** [comment]
Another vote for this on Wednesday.
